src/functions.php

```diff
@@ -67,6 +87,11 @@ function any(array $promisesOrValues)
         });
 }
 
+/**
+ * @param array $promisesOrValues
+ * @param $howMany
```

Comment:
This is the only place in react/promise where a type (`int`) makes sense because it is not `mixed` and not provided in the function/method signature.